elytron/WFCORE-4496-principal-transformer-in-aggregate-realm.adoc

```diff
@@ -0,0 +1,67 @@
+= [WFCORE-4496] Adding principal-transformer in aggregate-realm
+:author:            Ashley Abdel-Sayed
+:email:             aabdelsa@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+Elytron currently has an aggregate security realm which is a combination of two or more realms; an authentication realm
+and one or more authorization realms. There is currently no way to transform the principal in between loading the authentication
+identity and loading the authorization identity. This enhancement would allow transformations to be applied to the principal
+after loading the authentication identity and before loading the authorization identity.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFCORE-4496[WFCORE-4496]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1220[EAP7-1220]
```

Comment:
We should be focusing on the community issue here. So I think the WFCORE issue should stay in the ```Issue``` section and the EAP7 issue can be referenced in the ```Related Issues``` section.

Reply:
I am sorry, I missed that is the link to the WFCORE issue in the related links is the same as in the incorporates links.

Reply:
Fixed

Reply:
Just a minor comment - in general, for analysis documents, we have been referencing the EAP7 issue in the Related Issues section. Since this is a community project we should be prioritizing the use of community issues instead of using internal planning issues.

Reply:
Sorry @ashley-abdelsayed98 I did not catch this as it was happening.  The EAP7 issue must be in the related issues section only, the 'Issues' section is for the primary issues that the community can access.

Reply:
Fixed so EAP7 issue is under the related issues section

Reply:
Thanks @ashley-abdelsayed98